virttest/utils_disk.py

```diff
@@ -49,35 +49,61 @@ def copytree(src, dst, overwrite=True, ignore=''):
             shutil.copy(src_file, dst_dir)
 
 
-def is_mount(src, dst):
+def is_mount(src, dst, verbose=False, session=None):
     """
     Check is src or dst mounted.
 
     :param src: source device or directory, if None will skip to check
     :param dst: mountpoint, if None will skip to check
+    :param session: check within the session if given
 
     :return: if mounted mountpoint or device, else return False
```

Comment:
We can't know if the `src` is mounted to the `dst` by the current implementation, that's a problem.